### DISCUSSION TITLE
Does KubeEdge Support Routing Protocols?

### DISCUSSION THREAD

**Discussion by anjumm** [kubeedge/kubeedge#4292] [General]
Hi, hope everyone is good. I am learning about KubeEdge and would like to use this platform for IoT devices. I would like to know if we can use routing protocols such as BGP, MPLS and OSPF for communication between edge nodes? The reason for using these protocols is to employ traffic engineering. I will appreciate the quick response.

**Comment by vincentgoat** [comment]
Hi,
Do you mean traffic only between edge nodes? If so, I think it is like virtual machine communication, it will not be affected by KubeEdge.
